crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -905,6 +912,9 @@ impl HolochainP2pActor {
                 if !a.storage_arc.contains(loc) {
                     return None;
                 }
+                if local_agents.contains(&a.agent) {
```

Comment:
Filter local agents when getting peers near a location. Local agents should not be called over the network.